src/libraries/Microsoft.Extensions.Logging/gen/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Extensions.Logging.Generators
+{
+    public static class DiagnosticDescriptors
+    {
+        public static DiagnosticDescriptor InvalidLoggingMethodName { get; } = new (
```

Comment:
is the coding guideline for the `new` typeless constructor to have an space in between `new` and `(`?

Reply:
There was an issue with analyzer but it was fixed and `new()` works now.